packages/trader/src/AppV2/Components/TradeParameters/TradeTypeTabs/trade-type-tabs.tsx

```diff
@@ -12,9 +12,20 @@ type TTradeTypeTabsProps = {
 const TradeTypeTabs = observer(({ is_minimized }: TTradeTypeTabsProps) => {
     const { contract_type, onChange, trade_type_tab, setTradeTypeTab } = useTraderStore();
     const tab_list = getTradeTypeTabsList(contract_type);
-    const initial_index = tab_list.findIndex(tab =>
-        trade_type_tab ? tab.contract_type === trade_type_tab : tab.value === contract_type
-    );
+    let initial_index = 0;
+
+    // If the trade type tab is VANILLALONGPUT or TURBOSSHORT, keep the first tab
+    if (
+        !(
+            (trade_type_tab === 'VANILLALONGPUT' && contract_type === 'vanillalongcall') ||
```

Comment:
Nit: you may use `CONTRACT_TYPES` and `TRADE_TYPES` objects as we were trying to avoid direct typing such strings

Reply:
agree with that - hardcoded strings shouldbe minimalised,